plugin/etcd/plugin.go

```diff
@@ -45,76 +49,90 @@ func main() {
 		Fields: []plugin.Field{
 			plugin.Field{
 				Mode:     "target",
-				Name:     "etcd_url",
+				Name:     "url",
 				Type:     "string",
-				Title:    "Endpoint",
-				Help:     "It is the upstream etcd client endpoint",
-				Example:  "https://192.168.42.45:2379",
+				Title:    "Endpoints",
+				Help:     "The client URL(s) of the etcd cluster.",
+				Example:  "https://192.168.42.45:2379,https://192.168.23.54:2379",
 				Required: true,
 			},
 			plugin.Field{
-				Mode:    "target",
-				Name:    "auth",
-				Type:    "bool",
+				Mode:     "target",
+				Name:     "timeout",
+				Type:     "string",
+				Title:    "Dial Timeout",
+				Help:     "DialTimeout is the timeout for failing to establish a connection. Enter time in seconds.",
```

Comment:
Dial is a decidedly "Go" thing to say (thanks AT&T) - I would retitle as 'Connection Timeout' or just 'Timeout' and elude to the fact that it is timing out connections in the help text.